lib/webpacker/dev_server.rb

```diff
@@ -3,15 +3,19 @@ class Webpacker::DevServer
   # Webpacker.dev_server.connect_timeout = 1
   cattr_accessor(:connect_timeout) { 0.01 }
 
-  delegate :config, to: :@webpacker
+  delegate :config, :env, to: :@webpacker
 
   def initialize(webpacker)
     @webpacker = webpacker
   end
 
   def running?
-    Socket.tcp(host, port, connect_timeout: connect_timeout).close
-    true
+    if env.production?
+      false
+    else
+      Socket.tcp(host, port, connect_timeout: connect_timeout).close
+      true
+    end
```

Comment:
I think this should be inverted and check `env.development?` instead. We have production-like envs (`staging`, `beta`) that we want skipped too.
```ruby
if env.development?
  Socket.tcp(host, port, connect_timeout: connect_timeout).close
  true
else
  false
end
```

Reply:
All production like environments will have `NODE_ENV=production` and that will evaluate `env.production?` to `true` so the logic will work for all production like environments as long as NODE_ENV is set. Would that work? 

```rb
  def env
    (ENV["NODE_ENV"].presence_in(available_environments) ||
      Rails.env.presence_in(available_environments) ||
        "production".freeze).inquiry
  end
```

Reason for scoping to production was that, in case someone wants to use dev server in test environment. What do you think?

Reply:
Perhaps we can check if `dev_server` is configured? Since, by default, it's only in development: https://github.com/rails/webpacker/blob/6695445dc820b2f9f8adc354c3254a224c88747a/lib/install/config/webpacker.yml#L33-L38

Reply:
Ahh yes, good idea 👍

Reply:
Done 👍